frontend/src/components/InterviewPage.js

```diff
@@ -31,12 +31,29 @@ const InterviewPage = () => {
         setShowEndInterview(true)
     }
     const handleEndInterview = () => {
-        MatchingService.deleteInterview(interviewId)
-            .then((res) => {
-                setQuestionTitle("")
-                setQuestionDescription("")
-                setShowEndInterview(false)
-                navigate("/home")
+        MatchingService
+            .getInterview(interviewId)
+            .then(interviewDetails => {
+                console.log(interviewDetails)
+                if (interviewDetails.data === null) {
+                    console.log("Interview terminated from other user")
+                    setQuestionTitle("")
+                    setQuestionDescription("")
+                    setShowEndInterview(false)
+                    navigate("/home")
+                } else {
+                    MatchingService
+                        .deleteInterview(interviewId)
+                        .then((res) => {
+                            setQuestionTitle("")
```

Comment:
LGTM
However not sure if we should standardize using async/await vs using promises